warehouse/core/src/main/java/datawave/ingest/protobuf/TermWeightPosition.java

```diff
@@ -98,6 +100,11 @@ public boolean equals(TermWeightPosition o) {
         return (compareTo(o) == 0);
     }
     
+    @Override
+    public int hashCode() {
+        return System.identityHashCode(this);
```

Comment:
This kinda makes the hash code not useful.
```suggestion
        return new HashCodeBuilder().append(offset).append(prevSkips).append(score).append(zeroOffsetMatch).toHashCode();
```

Reply:
+1

Reply:
The hashcode I used was the SonarQube suggestion.  Switching it to this one.